brother_ql/cli.py

```diff
@@ -41,16 +44,50 @@ def cli(ctx, *args, **kwargs):
 @cli.command()
 @click.pass_context
 def discover(ctx):
-    """ find connected label printers """
-    backend = ctx.meta.get('BACKEND', 'pyusb')
-    discover_and_list_available_devices(backend)
+    """find connected label printers"""
+    backend = ctx.meta.get("BACKEND", "pyusb")
+    if backend is None:
+        logger.info("Defaulting to pyusb as backend for discovery")
+        backend = "pyusb"
+    from brother_ql.backends.helpers import discover, status
 
-def discover_and_list_available_devices(backend):
-    from brother_ql.backends.helpers import discover
     available_devices = discover(backend_identifier=backend)
-    from brother_ql.output_helpers import log_discovered_devices, textual_description_discovered_devices
-    log_discovered_devices(available_devices)
-    print(textual_description_discovered_devices(available_devices))
```

Comment:
Q: Can we leave this in?

Reply:
Is there a reason to remove it? I like to keep public functions around as the library is very widely used and ppl monkeypatch nearly everything public.

Reply:
Fine, I'll add the function back in.